LICENSE

```diff
@@ -631,8 +631,8 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    <one line to give the program's name and a brief idea of what it does.>
-    Copyright (C) <year>  <name of author>
+    {one line to give the program's name and a brief idea of what it does.}
```

Comment:
Should I fill this in? How about *OpenMW, free and open-source Morrowind engine reimplementation*

Reply:
Or better yet *engine **and editor***.

Reply:
I feel like a lawyer now.

Reply:
I'll give @psi29a a ping since pretending to be a lawyer is one of his expertises ;)

Reply:
Your description works for me.